src/config.js

```diff
@@ -5,7 +5,7 @@ const config = Object.freeze({
   port: process.env.TR_PORT || '3000',
   sessionSecret: process.env.TR_SESSION_SECRET || 'override_this_value',
   apiEndpoint: process.env.TR_API_URL || 'http://localhost:3001/trap-registration-api/v2/registrations',
```

Comment:
This should be `'http://localhost:3001/trap-registration-api'` - I'm not sure why you changed it back?

You should add the `/v1` or `/v2` when an axios call is made. Otherwise you can only ever call `/v2/registrations` when using the `apiEndpoint` in your axios calls.